WordPress/Classes/ViewRelated/NUX/CreateAccountAndBlogViewController.m

```diff
@@ -699,8 +699,6 @@ - (void)createUserAndSite
     WPAsyncBlockOperation *userCreation = [WPAsyncBlockOperation operationWithBlock:^(WPAsyncBlockOperation *operation){
         void (^createUserSuccess)(id) = ^(id responseObject){
             // Turn on the new editor only for users that create a new account within the iOS app
-            [WPPostViewController setNewEditorAvailable:YES];
-            [WPPostViewController setNewEditorEnabled:YES];
             [WPAnalytics track:WPAnalyticsStatEditorEnabledNewVersion];
```

Comment:
@diegoreymendez @sendhil given what @meremagee said above, should we just add  ```[WPAnalytics track:WPAnalyticsStatEditorEnabledNewVersion];``` to ```setNewEditorEnabled``` in ```WPPostViewController```? And...do we need a new item in the ```WPAnalyticsStat``` enum to capture when the visual editor is turned off?